packages/utils/src/token.ts

```diff
@@ -12,3 +13,12 @@ export function isAlias(value: unknown): boolean {
   if (typeof value !== 'string') return false;
   return ALIAS_RE.test(value);
 }
+
+/** get last segment of a token ID */
+export function getLocalID(id: string): string {
```

Comment:
This is such a random little utility I didn’t realize I’d be needing as often as I do—getting the last segment of an ID path.